src/components/HighlightWord.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { FC } from "react"
-import { RoleTemplateProps } from "../templates/home-page/SectionTemplate"
-import { BoxProps, Text } from "@chakra-ui/react"
+import { Box, BoxProps, Text } from "@chakra-ui/react"
```

Comment:
Let's remove `Box` import from here

Reply:
Addressed in [Remove unused box element import](https://github.com/threshold-network/website/pull/105/commits/2af0b4d60eaeb72f830b13d42c90d84dbb78cb4d)